pkg/daemon/ceph/osd/volume.go

```diff
@@ -866,7 +868,7 @@ func GetCephVolumeLVMOSDs(context *clusterd.Context, clusterInfo *client.Cluster
 			SkipLVRelease: skipLVRelease,
 			LVBackedPV:    lvBackedPV,
 			CVMode:        cvMode,
-			Store:         "bluestore",
+			Store:         os.Getenv(oposd.OSDStoreTypeVarName),
```

Comment:
How are we handling the case where this is reading an existing OSD? Empty should read as `"bluestore"`

Reply:
Using Empty as `bluestore`. 

Regarding existing OSDs, I believe we are covered there.  We won't be allowing changing the OSD store in the spec without allowing the migration of OSDs (one by one).  Once the OSD will be migrating, no other OSD deployment will be allowed to upgrade. So existing OSDs that might be user legacy store will not be affected by this change.

Reply:
Good point!